appcoins-ads/src/main/java/com/asf/appcoins/sdk/ads/repository/ResponseCode.java

```diff
@@ -0,0 +1,17 @@
+package com.asf.appcoins.sdk.ads.repository;
+
+public enum ResponseCode {
+  BILLING_UNAVAILABLE(3), DEVELOPER_ERROR(5), ERROR(6), FEATURE_NOT_SUPPORTED(
```

Comment:
WE have many response codes that don't make sense in this module, please remove the not used